CHANGELOG.md

```diff
@@ -4,7 +4,16 @@
 
 ## New features
 
-- [#2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)
+#### Allow users to use cookie banner buttons as links
+
+You can now add links styled as buttons to cookie banners.
+
+Use this feature only if you need to. For example, to display the **Hide** button because:
+
+- your cookie banner needs to work without JavaScript
```

Comment:
Does the **Hide** button not display for non-JS users unless they style links as buttons?